ogzaf/macro_params.py

```diff
@@ -114,5 +216,5 @@ def get_macro_params():
     res = mod.fit()
     macro_parameters["r_gov_shift"] = res.params["BAA Corp Bond Rates"]
     macro_parameters["r_gov_scale"] = res.params["constant"]
```

Comment:
I think I have a solution that uses global data to estimate the scale and shift parameters. I'll open an issue and propose a PR

Reply:
thank you @SeaCelo . Are you thinking about a separate PR to the main or a PR to this branch?

Reply:
I’ll try to do one to this branch if I get it right. It’s a small code change but I want to validate the approaches first. I’ll create an issue to discuss it